custom/icds_reports/ucr/reports/testing/mpr_10a_children_referred.json

```diff
@@ -101,7 +101,7 @@
         "aggregation": "sum",
         "calculate_total": true,
         "whens": {
-          "referral_health_problem ~ '\ypremature\y'": 1
+          "referral_health_problem ~ '\\ypremature\\y'": 1
```

Comment:
what _is_ the `\y` character for?  Some kind of quote?

Reply:
oof. I thought that I included that in the PR description. That's postgres's syntax for word boundary https://www.postgresql.org/docs/9.6/static/functions-matching.html#POSIX-CONSTRAINT-ESCAPES-TABLE